semgrep/semgrep/cli.py

```diff
@@ -324,6 +324,11 @@ def cli() -> None:
             "containing a 'nosem' comment at the end."
         ),
     )
+    output.add_argument(
+        "--experimental",
```

Comment:
Can we name this something less generic like `--core-direct-rules`?

Reply:
I think it's probably fine; this is really only meant for internal use. I think Pad was thinking that this would be an entrypoint for future experimental work, too. In any case, we can always change it later - it's just a string after all.

Reply:
yep, --core-direct-rules sounds good.